miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/ApiClient.kt

```diff
@@ -111,7 +111,7 @@ internal class RetrofitRequestExecutor(
         }
     }
 
-    @Throws(MiniAppSdkException::class)
+    @Throws(MiniAppSdkException::class, MiniAppNotFoundException::class)
```

Comment:
`MiniAppNotFoundException` is `MiniAppSdkException` so I don't think we need to add it here.

Reply:
I have followed the way like `MiniApp.create` where are keeping both. I think, the main reason is distinguish both as separate exception while providing error message.